varia/sass/blocks/cover/_editor.scss

```diff
@@ -3,7 +3,10 @@
 
 	background-color: #{map-deep-get($config-cover, "color", "background")};
 	color: #{map-deep-get($config-cover, "color", "foreground")};
-	min-height: #{map-deep-get($config-cover, "height")};
+
+	.block-library-cover__resize-container:not([style*="min-height"]) & {
```

Comment:
What is this intending? Is there any case where `.block-library-cover__resize-container` does not have the inline style?

Reply:
Ahh, after looking again, it seems this selector is probably not necessary. It was intended to only catch blocks that were made before 6.4, but you’re right that it doesn’t actually do that. I do think its necessary to prevent disruption for Cover Blocks that were made before 6.4. For those, we want the theme-specific height variable to still work.

Reply:
Yes, I guessed that, but I was confused because it looks like all cover blocks regardless when it was made seem to have the `min-height: 430px` in the editor after updating 6.4.